libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -4439,6 +4608,8 @@ export interface IDashboardWidgetProps {
     onFiltersChange?: (filters: (IDashboardFilter | FilterContextItem)[], resetOthers?: boolean) => void;
     // @alpha
     onWidgetClicked?: () => void;
+    // @alpha
+    parentLayoutItem?: IDashboardLayoutItemFacade<ExtendedDashboardWidget>;
```

Comment:
Not sure if it's good idea to pass `IDashboardLayoutItemFacade` here as it exposes all the stuff above. Maybe just pass the size as it is the only prop I need from this object?